lib/dao.js

```diff
@@ -2426,7 +2470,19 @@ DataAccessObject.prototype.updateAttributes = function updateAttributes(data, op
     inst.setAttributes(data);
 
     inst.isValid(function (valid) {
-      if (!valid) {
+
```

Comment:
At first I did not call the `isValid`-function, when no validation was needed. But then some test concerning the lifecycle hooks failed, because the `beforeValidation` and `afterValidation` hooks where missing. I was not sure if I could modify those, so instead I always validate, and depending on the settings and options I throw a ValidationError or I don't